code/game/objects/items/weapons/vending_items.dm

```diff
@@ -69,5 +69,5 @@
 /obj/item/weapon/vending_refill/clothing
 	machine_name = "ClothesMate"
 	icon_state = "refill_clothes"
-	charges = list(30, 2, 3)// of 86 standard, 6 contraband, 9 premium
-	init_charges = list(30, 2, 3)
\ No newline at end of file
+	charges = list(31, 2, 4)// of 87 standard, 6 contraband, 10 premium(?)

```

Comment:
i only did that because the comment is likely innacurate now. i wasn't sure if the 87 standard, etc, meant the number of unique products in the machine (counting one of each) or the total number of products (counting all the products + any duplicates, such as leather jackets, which are in stock of 3 and therefore have 2 duplicates)

no it wont affect the code since it's in a comment